src/components/dropdown-filter-ajax/dropdown-filter-ajax.js

```diff
@@ -246,12 +246,13 @@ class DropdownFilterAjax extends DropdownFilter {
    */
   getData = (query = '', page = 1) => {
     this.setState({ requesting: true });
-    Request
+    const request = Request
       .get(this.props.path)
       .query(this.getParams(query, page))
       .query(this.props.additionalRequestParams)
       .set('Accept', this.props.acceptHeader)
       .end(this.ajaxUpdateList);
+    this.setState({ request });
```

Comment:
I don't think this should be stored in state. Can discuss with you if you want to go over what you're trying to achieve.